internal/installer/apt.go

```diff
@@ -0,0 +1,131 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Wdyt about renaming the file to `deb` (the binary extension)? We are already doing the same for rpm and tar. Otherwise we are doing an abstraction about the package manager (apt) where for RPM we are not using yum